src/installation/upgrading.md

```diff
@@ -1,43 +1,45 @@
 # Upgrading from a previous version
 
-To upgrade Erigon to a newer version when you've originally installed it via Git and manual compilation, you should follow these steps without needing to delete the entire folder:
+Updating to the latest version of Erigon gives you access to the latest features and ensures optimal performance and stability.
 
-* **Terminate your Erigon** session by pressing CTRL+C
+## General Recommendations Before Upgrade
 
-* **Navigate to your Erigon directory**
+- **Read Release Notes**: Carefully review the [Release Notes](https://github.com/erigontech/erigon/releases) for breaking changes and new features relevant to your setup.
+- **Terminate your Erigon**: End your current Erigon session by pressing `CTRL+C`.
+- **Backup**: Always back up your `datadir` before performing major upgrades.  
 
-* **Fetch the latest changes from the repository**: You need to make sure your local repository is up-to-date with the main GitHub repository. Run:
+## Snapshots Format
 
-    ```bash
-    git fetch --tags
-    ```
+As of Erigon 3.1 Pebble Paws, the snapshot format has been updated so that you will no longer need to perform data upgrades or re-sync from scratch with future releases.
```

Comment:
`you will no longer need to perform data upgrades` - wrong. in previous releases updating data was optional - in 3.1 nothing changed - it's still optional. but if user want (to get all our data-fixes and disk-format changes) they now have tool to upgrade data: `erigon snapshots reset` - it will force Erigon to download changed files (and keep unchanged).